frontend/src/metabase/components/SingleSelectListField/SingleSelectListField.styled.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from "@emotion/styled";
```

Comment:
I lazily did a copy+paste job from ListField to SingleSelectListField. Perhaps there's some room for reuse here. The good thing is all the changes are in one commit [here](https://github.com/metabase/metabase/pull/25960/commits/d05aa0b24c25598bcc8566f43bb659a6f5175213), so finding opportunities for reuse is easy.